upgrade-notes/backend_20241007_110001.md

```diff
@@ -0,0 +1,5 @@
+#### enable Convertim usage on your project ([#{pullRequestId}](https://github.com/shopsys/shopsys/pull/{pullRequestId}))
+
+-   `frontend-api` `Cart` object now has property `promoCodes (array)` instead of current `promoCode (string)` and includes additional data about promo codes
+-   multiple constructor interfaces have been changed, review `PHP errors` and `standards` and fix possible issues
```

Comment:
I do not think we need to use this sentence, I hope we will put some general info at the beginning of the upgrade notes